shared/overview/2.x.md

```diff
@@ -4,52 +4,55 @@
+{{ $names.os.upper }} is an operating system built for easy portability to multiple device types (via the [Yocto framework][yocto] and optimized for Linux containers, and Docker in particular. There are many decisions, large and small, we have made to enable that vision, which are present throughout our architecture.
 
 The first version of {{ $names.os.lower }} was developed as part of the {{ $names.company.lower }} platform, and has run on thousands of embedded devices on {{ $names.company.lower }}, deployed in many different contexts for several years. {{ $names.os.lower }} v2 represents the combination of the learnings we extracted over those years, as well as our determination to make {{ $names.os.lower }} a first-class open source project, able to run as an independent operating system, for any context where embedded devices and containers intersect.
 
 We look forward to working with the community to grow and mature {{ $names.os.lower }} into an operating system with even broader device support, a broader operating envelope, and as always, taking advantage of the most modern developments in security and reliability.
 
 ## Variants of {{ $names.os.lower }}
 
-{{ $names.os.upper }} currently comes in 3 different variants all built from the same source but with slightly differing features enabled or disabled. Each version of {{ $names.os.lower }} produces the following variants:
+Each version of {{ $names.os.lower }} is available in development and production variants, both built from the same source, but with slightly differing feature sets.
 
 | Version Name | Variant Type | Description |
-|:------------:|:------------:|:------------:|
-| 2.0.0+rev2   | production   | This is the production version of the {{ $names.company.lower }} managed OS. This is the OS you should use for any production fleet deployments |
-| 2.0.0+rev2-dev | development | This version the development version of the above and should be used when you are developing a new application and want to use the fast [local mode][local-mode-link] workflow. This variant should never be used in production. |
-| 2.0.0+rev2.dev | standalone | This version is an unmanaged version of {{ $names.os.lower }}, it doesn't have the {{ $names.company.lower }} supervisor and does not connect to the management console. Use this variant if you want a stable linux OS to run Docker on. |
+|:----------------:|:------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+| 2.46.1+rev3-prod | production | This is the production version of the {{ $names.company.lower }} managed OS. This is the OS you should use for any production fleet deployments |
```

Comment:
This PR removes the "unmanaged version" from this table. Does that mean the "managed OS" here is / isn't relevant? :man_shrugging:

Reply:
👍 Thanks for this, I left this table unchanged (save for deleting that row) so I've now cleaned up the terminology here for consistency.